var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -59,10 +59,17 @@ class Hypre(Package):
     # Patch to add ppc64le in config.guess
     patch('ibm-ppc64le.patch', when='@:2.11.1')
 
+    # Patch to build shared libraries on Darwin
+    patch('darwin-shared-libs.patch', when='@2.13.0:')
```

Comment:
you could restrict this to only `+shared` and maybe only on darwin

Reply:
Sure.